mongodb-atlas/mongodbatlasplugin/mongodb-atlas-config.go

```diff
@@ -0,0 +1,6 @@
+package mongodbatlas
+
+type MongoDBAtlasConfig struct {
+	PublicKey  string
```

Comment:
I'd go ahead and get this reading off the file system during the boot sequence. Check out how we do it in `getDatadogConfig` and `getConfigFilePath()` in that plugin. This config will grow quickly as this matures and you will be glad to get it in a file ASAP before you get trapped in env var hell world